apiserver/controllers/controllers.go

```diff
@@ -110,14 +145,23 @@ func (a *APIController) handleWorkflowJobEvent(w http.ResponseWriter, r *http.Re
 	signature := r.Header.Get("X-Hub-Signature-256")
 	hookType := r.Header.Get("X-Github-Hook-Installation-Target-Type")
 
+	hostname, controllerId := a.GetControllerInfo()
```

Comment:
in case of `hostname` I disagree a little bit. The hostname can change at any time, it's not in our hands. I would be nice if the metrics would reflect this.